portfolio/src/main/webapp/game.js

```diff
@@ -9,7 +9,23 @@ class Board {
       [0, 0, 0]
     ];
     this.drawGrid();
-    var red = true; 
+    this.red = true; 
+    this.numTiles = 0;
+  }
+
+  /* 
+  * Initalizes game
+  */
+  init() {
+    this.board = [
+      [0, 0, 0],
+      [0, 0, 0],
+      [0, 0, 0]
+    ];
+    this.red = true; 
+    this.numTiles = 0;
+    ctx.clearRect(0, 0, canvas.width, canvas.height);
```

Comment:
(optional) - This is borderline but I would consider moving this to a clearBoard() function.